selfservice/flow/login/handler.go

```diff
@@ -250,6 +264,12 @@ type createNativeLoginFlow struct {
 	//
 	// in: header
 	SessionToken string `json:"X-Session-Token"`
+
+	// EnableSessionTokenExchangeCode requests the login flow to include a code that can be used to retrieve the session token
+	// after the login flow has been completed.
+	//
+	// in: query
+	EnableSessionTokenExchangeCode bool `json:"enable_session_token_exchange_code"`
```

Comment:
Is this parameter needed?

Reply:
Yes, so that the user can opt in to the code-specific flow behaviour.

Reply:
No we can't, because the code-specific flow behaviour is different from the non-code behaviour.

When issuing a code, we redirect the user to the `return_to` address so that the app gets back control and can retrieve the session token from the code.

When *not* issuing the code, we return the session token directly as JSON.